blog/2024-01-15-statecharts-whats-in-a-name/index.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Statecharts - What’s in a name?
```

Comment:
Sure, sounds good.

Reply:
I've done most of the renaming to "state machine". Some references to the publications still need to remain "statecharts" and I'll save renaming the blog folder and slug until the end so that it doesn't invalidate any other feedback on this PR.

Reply:
All renaming is done and I updated the date of the post to be tomorrow.